packages/react-app/src/models/Bounty.ts

```diff
@@ -7,6 +7,10 @@ import {
 	array,
 	mixed,
 } from 'yup';
+
+// Have to use ignore here due to some strange error with typescript not picking
+// up the declaration file, unless you change the name every deploy
+// @ts-ignore
```

Comment:
I thought the `ban-ts-annotation` config set earlier in this PR would disallow this?

https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/ban-ts-comment.md

Reply:
Disabled the setting.